proxy/coco/src/ethereum_claim_ext.rs

```diff
@@ -0,0 +1,32 @@
+//! The user identity doc extension for Ethereum addresses attestation.
+//! See [the RFC](docs/ethereum_attestation.md).
+
+use lazy_static::lazy_static;
+use librad::identities::payload::HasNamespace;
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+/// The user identity doc extension for Ethereum addresses claims.
+/// Meaningful only if confirmed on Ethereum, see [the RFC](docs/ethereum_attestation.md).
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct V1 {
+    /// The Ethereum address claimed by the user, encoded according to
+    /// [EIP-55](https://eips.ethereum.org/EIPS/eip-55).
+    pub address: String,
```

Comment:
I'd suggest a newtype and parser to make sure the construction is always compliant with EIP-55

Reply:
Fixed